scripts/imgtool/image.py

```diff
@@ -51,6 +51,7 @@
         'PIC':                   0x0000001,
         'NON_BOOTABLE':          0x0000010,
         'RAM_LOAD':              0x0000020,
+        'ROM_FIXED':             0x0000100,
         'ENCRYPTED':             0x0000004,
```

Comment:
Since you are not adding the new value at the end, also sort this one into the correct position.